src/rules/switchIdentifiersRule.ts

```diff
@@ -37,42 +37,44 @@ export class SwitchIdentifiersWalker extends ProgramAwareRuleWalker {
 
   /** Method that is called for every identifier inside of the specified project. */
   visitIdentifier(identifier: ts.Identifier) {
-    // In some situations, developers will import Angular Material completely using a namespace
-    // import. This is not recommended, but should be still handled in the migration tool.
-    if (isNamespaceImportNode(identifier) && isMaterialImportDeclaration(identifier)) {
-      const symbol = getOriginalSymbolFromNode(identifier, this.getTypeChecker());
-      return this.materialNamespaceDeclarations.push(symbol.valueDeclaration);
-    }
+    // Store Angular Material namespace identifers in a list of declarations.
```

Comment:
Can you add a comment with an example here for anyone not clear that "namespace identifiers" means?

Reply:
Done. Added a small comment with an example.